keycloak/helm/keycloak/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
could you rm this file?  It's normally not helpful and can make helm installs flake.